### DISCUSSION TITLE
Console log

### DISCUSSION THREAD

**Discussion by valpuia** [malzariey/filament-daterangepicker-filter#22] [Q&A]
![Screenshot from 2023-06-06 09-58-20](https://github.com/malzariey/filament-daterangepicker-filter/assets/28250303/d8848633-1ce5-4711-baf4-3799ca52acd2)

In console `move` is logged.. and when clearing the filter `cannot read proper...` comes

I don't wanna raise an issue because of console log, so I post here

`Upgrading malzariey/filament-daterangepicker-filter (1.2.1 => 1.3)`

**Comment by malzariey** [comment]
Thanks @Valpuia, please check the latest version.
